cloudformation/infrastructure/rds.yaml

```diff
@@ -42,10 +46,10 @@ Resources:
       StorageType: gp2
       BackupRetentionPeriod: '7'
       MasterUsername: concordia
-      MasterUserPassword: MyPassword
+      MasterUserPassword: !Ref DbPassword
```

Comment:
We shouldn't store the password here where it could potentially be read by someone with CloudFormation access. Fortunately, [there are several options for this](https://serverfault.com/a/930755/4603), and I'd recommend  ([SSM secure parameters](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/dynamic-references.html)) which I used for the shared Sentry deployment. 

```
"MasterUserPassword": "{{resolve:ssm-secure:/devops/sentry/DB/MasterUserPassword:1}}",
```

Storing that requires two steps:

1. Setup a KMS key with an alias like `concordia-production`: https://console.aws.amazon.com/iam/home?#/encryptionKeys/us-east-1/
2. Generate and store the password: `aws ssm put-parameter --type SecureString --key-id "alias/concordia-production" --name "/concordia/production/DB/MasterUserPassword" --value $(pwgen -s 30 1)`

There's one catch to this: currently, [ECS does not have a supported mechanism to retrieve SSM parameters](https://github.com/aws/amazon-ecs-agent/issues/1209). That means that the app would need to either use boto to retrieve them or have its Docker entrypoint retrieve it before starting, perhaps something like this:

```
DB_PASSWORD="$(aws ssm get-parameter --output=text --name /concordia/production/DB/MasterUserPassword --with-decryption --query "Parameter.Value")"
export DB_PASSWORD
```